spec/classroom_spec.rb

```diff
@@ -0,0 +1,10 @@
+require './classroom'
+
+describe Classroom do
+  classroom = Classroom.new('Math')
+  context 'should create instance of classroom class' do
+    it 'returns correct label' do
+      expect(classroom.label).to eq('Math')
+    end
```

Comment:
- [ ] it would be better to add tests for `add_student` and also for `can_use_services` in order to cover all the methods for `can_use_services` you should test positive case and negative case

Reply:
@utsargo sorry I just forgot to remove it